pom.xml

```diff
@@ -34,10 +34,10 @@
             <id>spigot-repo</id>
             <url>https://hub.spigotmc.org/nexus/content/groups/public</url>
         </repository>
-        <repository>
+        <!--<repository>
```

Comment:
New repository is hosted at http://nexus.hc.to/content/repositories/pub_releases. If you just comment it out, probably will just stop building.